### DISCUSSION TITLE
#45 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Emilyxiannv** [tog-ETH-er/togETHer#23] [General]
Agenda:

Community updates
Fireside discussion

> 会议主题：togETHer
> 会议时间： 21:00-22:00
> 重复周期：每两周 (周一)
> 
> 点击链接入会，或添加至会议列表：
> https://meeting.tencent.com/s/LxNajKWJw4RV
> 
> 会议 ID：891 5098 6775

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 官网翻译 review 进行中

**Comment by stephanieeee-ho** [comment]
### ECN
1. 常规的翻译和七日谈
2. 大理以太坊主题营地活动的讨论

**Comment by luckyyang** [comment]
### Rebase
#### [招聘](https://github.com/rebase-network/who-is-hiring/issues)
![2022-08-08 at 20 27](https://user-images.githubusercontent.com/1963646/183417785-b3429f82-4ddd-49d0-9f34-ec81934a9f57.png)

#### [Web3 Daily](https://rebase.network/posts/category/web3daily)
更新 14 篇日报

#### [活动](https://github.com/rebase-network/event-pool/issues)
支持了 5 个活动
![2022-08-08 at 20 32](https://user-images.githubusercontent.com/1963646/183418701-bf24e3c2-0eec-4916-ba31-ebc66f14fcfe.png)

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

4个PR #811 ～ #814

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了1场分享会

- 《Polynomial Commitment (KZG) with Examples》 by qizhou
  - https://www.youtube.com/watch?v=n4eiiCDhTes

- 《lens协议分享》by Tao

## 翻译小组

- MIT公开课正式发布第21集
- 第22,23 审稿中

**Comment by Thegaram** [comment]
## Crypto Tech Night

### CTN #16

08.04: CTN #16 DeFi Under The Hood 深入探索DeFi的“代码世界” (\w DappLearning)
- 0xstan: Euler Finance
- 七哥: Evabase

回放：https://bilibili.com/video/BV1VS4y1s7UP

--

### 未来的规划

CTN新加入了一位美国的朋友，团队以后也越来越去中心化，所以我们准备开始做更多的英文的分享，包括Twitter content和技术分享活动。

**Comment by Stamp9** [comment]
### 原语里弄
瓦猫之夏活动协助

**Comment by bobjiang** [comment]
- Gitcoin GR15 Sep 7 - Sep 22
- Gitcoin grant matching fund claim
- Gitcoin中文周报 http://news.bobjiang.com/
